siteminder.module

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @file
+ * Siteminder authentication module for Drupal.
+ * 
+ */
+
+use Drupal\Core\Url;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_help().
+ */
+function siteminder_help($route_name) {
+  switch ($route_name) {
+    case 'siteminder.admin_settings':
+    case 'help.page.siteminder':
+      $output = t('<p>This module integrates Drupal with a Siteminder Agent running on the webserver.</p>');
+
+      return $output;
+  }
+}
+
+/**
+ * Implements hook_user_logout().
+ */
+function siteminder_user_logout($account) {
+  //Destory Drupal Session and remove the cookie from the browser set by the siteminder
+  //Redirect the user to the Siteminder agent to log-off.
+  $logout_url = \Drupal::config('siteminder.settings')->get('logout_url');
+  $siteminder_cookie = \Drupal::config('siteminder.settings')->get('siteminder_cookie');
+  $siteminder = \Drupal::service('siteminder.siteminderhelper');    
+  session_destroy();
```

Comment:
hmm, it doesn't sound right. The session handler in core takes care of destroy. are we sure we should?